packages/widget/src/interfaces/index.ts

```diff
@@ -0,0 +1,30 @@
+import type {
+  EvmResource,
+  Network,
+  SubstrateResource
+} from '@buildwithsygma/sygma-sdk-core';
+import type { ethers } from 'ethers';
+import type { ApiPromise } from '@polkadot/api';
+import type { Signer } from '@polkadot/api/types';
+
+export type ThemeVariables =
+  | 'mainColor'
+  | 'secondaryColor'
+  | 'fontSize'
+  | 'borderRadius'
+  | 'borderRadiusSecondary';
+
+export type Theme = Record<ThemeVariables, string>;
+
+export interface ISygmaProtocolWidget {
+  whitelistedNetworks?: Network[];
+  evmProvider?: ethers.providers.Web3Provider;
```

Comment:
it should be https://github.com/ethers-io/ethers.js/blob/a26ff77c210660e641455150ccea6c0530f02f61/src.ts/providers/provider-browser.ts#L16

Also instead of depending on ethers package in this react package, you can just c/p this interface